nx-plugin/src/generators/search/files/ngrx/src/app/__featureFileName__/pages/__featureFileName__-search/__featureFileName__-search.effects.ts.template

```
@@ -272,7 +329,7 @@ updateSearchConfig$ = createEffect(() => {
       key: '<%= featureConstantName %>_SEARCH.ERROR_MESSAGES.SEARCH_CONFIG_UPDATE_FAILED',
     },
     {
-      action: <%= featureClassName %>SearchActions.searchConfigsLoadingFailed,
+      action: <%= featureClassName %>SearchActions.searchConfigInfosLoadingFailed,
```

We said that we wanted to remove this action from the list so it is failing silently in case of the endpoints are not implemented. In case of an error, the feature silently gets not activated.